Client/Frontend/Browser/Tab.swift

```diff
@@ -197,6 +197,8 @@ class Tab: NSObject {
             configuration!.allowsInlineMediaPlayback = true
             // Enables Zoom in website by ignoring their javascript based viewport Scale limits.
             configuration!.ignoresViewportScaleLimits = true
+            // Enables media auto-play based on the current preference setting (default is false)
+            configuration!.mediaTypesRequiringUserActionForPlayback = Preferences.General.mediaAutoPlays.value ? [] : .all
```

Comment:
Doing this here is not required since, this is set when in `getNewConfiguration()`